Yona/Pods/Target Support Files/Pods-Yona/Pods-Yona-acknowledgements.markdown

```diff
@@ -1,6 +1,14 @@
 # Acknowledgements
 This application makes use of the following third party libraries:
 
+## Crashlytics
+
+Fabric: Copyright 2018 Google, Inc. All Rights Reserved. Use of this software is subject to the terms and conditions of the Fabric Software and Services Agreement located at https://fabric.io/terms. Crashlytics Kit: Copyright 2018 Crashlytics, Inc. All Rights Reserved. Use of this software is subject to the terms and conditions of the Crashlytics Terms of Service located at http://try.crashlytics.com/terms/terms-of-service.pdf and the Crashlytics Privacy Policy located at http://try.crashlytics.com/terms/privacy-policy.pdf. OSS: http://get.fabric.io/terms/opensource.txt
```

Comment:
Should this line start with `Fabric: `?

Reply:
Yes, as Crashlytics is part or Fabric.

Reply:
Discussed, these lines are auto-generated during pod installation so should be correct. Crashlytics is part of Fabric.